### ISSUE TITLE
Could you provide a model and parameters to reproduce the davidwu effect in the paper?

### ISSUE THREAD

**Issue by XiaoluDing** [drboog/ProFusion#5]
Amazing effect!! could you provide a model and parameters to reproduce the davidwu effect in the paper, plz

![image](https://github.com/drboog/ProFusion/assets/12556657/6c9443ba-8d54-4038-ae99-cecde20cc645)

**Comment by drboog** [comment]
I didn't save weights of that fine-tuned model, but you can try to fine-tune the model by your self. You can try slightly larger batch size and more iteration steps to ensure good details, then tune the hyper-parameters of fusion sampling at testing.

**Comment by ywdong** [comment]
> I didn't save weights of that fine-tuned model, but you can try to fine-tune the model by your self. You can try slightly larger batch size and more iteration steps to ensure good details, then tune the hyper-parameters of fusion sampling at testing.

Which hyper-parameters of fusion is important? I test my own pic, with bs=2, iteration=100, other hyper-parameters is the same  with your test code. But the image is not so good.

**Comment by drboog** [comment]
First of all, you need to make sure that your details is well-preserved after fine-tuning, i.e. "a photo of S*" can generate expected details. Otherwise, you may want to change batch size, learning rate, and iteration number. I think you should try more iterations, if your batch size is 2.

After fine-tuning, please read the comments in the example, different hyper-parameters control different things. Basically, tune cfg, ref_cfg and refine_emb_scale. How to tune them are based on what your current generation looks like. You may want to use a loop to generate multiple images for a hyper-parameter combination, to better understand and tune it.
